llvm/test/Transforms/LoopVectorize/tail-folding-live-out-scalar-vf.ll

```diff
@@ -0,0 +1,53 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --version 6
+; RUN: opt -p loop-vectorize -prefer-predicate-over-epilogue=predicate-else-scalar-epilogue -force-vector-width=1 -force-vector-interleave=2 -S %s | FileCheck %s
+
+define i64 @live_out_scalar_vf(i64 %n) {
```

Comment:
if possible, might be good to add this to `llvm/test/Transforms/LoopVectorize/tail-folding-vectorization-factor-1.ll`, which already contains a set of test cases for interleaving only with tail-folding.